packages/block-library/src/comments-pagination-previous/index.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Server-side rendering of the `core/comments-pagination-previous` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/comments-pagination-previous` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the previous posts link for the comments pagination.
+ */
+function render_block_core_comments_pagination_previous( $attributes, $content, $block ) {
+	$default_label    = __( 'Previous Comments' );
+	$label            = isset( $attributes['label'] ) && ! empty( $attributes['label'] ) ? $attributes['label'] : $default_label;
+	$pagination_arrow = get_comments_pagination_arrow( $block, 'previous' );
+	if ( $pagination_arrow ) {
+		$label = $pagination_arrow . $label;
+	}
+
+	$filter_link_attributes = function() {
+		return get_block_wrapper_attributes();
+	};
+	add_filter( 'previous_comments_link_attributes', $filter_link_attributes );
+
+	$previous_comments_link = get_previous_comments_link( $label );
+
+	remove_filter( 'previous_comments_link_attributes', $filter_link_attributes );
+
+	if ( ! isset( $previous_comments_link ) ) {
```

Comment:
Nit: Do we need this check?

Reply:
`get_previous_link()` returns void if we are on the first page.

We are assigning a void return to a variable (not sure if that's the best approach), and then we would return it if we don't do that check.

I think we are returning an empty string in order to avoid returning two different types in the same function.

Reply:
I think it doesn't really matter as the end result will be the same(nothing rendered) but it's minor in general 😄